internal/server/server.go

```diff
@@ -3,28 +3,27 @@ package server
 import (
 	"context"
 	"fmt"
-	newRelic2 "github.com/goto/stencil/pkg/newrelic"
 	"log"
 	"net/http"
 	"strings"
 
+	"github.com/goto/stencil/core/changedetector"
```

Comment:
looks like some lint issue

Why this import is not grouped in the line number 65?

Reply:
Try make lint? and if that is not working as expected then we can recheck this later.

Reply:
did make lint. Nothing happened in this file. Few got impacted.